ffconv/process.py

```diff
@@ -311,10 +311,13 @@ def clean_up(self):
         pass
 
     def convert(self):
-        cmd = ['ffmpeg', '-i', self.input, '-map', '0:{}'.format(self.index),
-               '-c:a', self.target_codec, '-q:a', str(self.target_quality),
-               '-ac:0', str(self.target_channels), self.output]
-        execute_cmd(cmd)
+        if True:
+            if True:
+                if True:
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Merge this if statement with the enclosing one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://localhost:9000/coding_rules#rule_key=python%3AS1066)